### ISSUE TITLE
Layer versions >104 not available in eu-north-1

### ISSUE THREAD

**Issue by tibbing** [DataDog/datadog-lambda-js#495]
## Expected Behavior
Layer versions >104 available in all regions

## Actual Behavior
Layer versions 105 and 106 are missing in `eu-north-1`, perhaps other regions as well. These versions are however available in `eu-west-1` and `eu-central-1`. Layer versions up to 104 do exist in `eu-north-1`.

Is there a list of supported regions somewhere?

## Steps to Reproduce the Problem

```bash
aws lambda get-layer-version \
  --region eu-north-1 \
  --layer-name arn:aws:lambda:eu-north-1:464622532012:layer:Datadog-Node18-x \
  --version-number 106
```
Returns:
```bash
An error occurred (AccessDeniedException) when calling the GetLayerVersion operation: User: arn:aws:sts::xxxxxxxx:xxxxxxxxxx is not authorized to perform: lambda:GetLayerVersion on resource: arn:aws:lambda:eu-north-1:464622532012:layer:Datadog-Node18-x:106 because no resource-based policy allows the lambda:GetLayerVersion action
```

## Specifications

  - Datadog Lambda Layer version: 106
  - Node version: 18

**Comment by astuyve** [comment]
Hey @tibbing - thanks for reaching out!
Looks like eu-north-1 was erroneously omitted from our regions list as we've moved to a ci-based release process. I'll get this cleared up right away.

Thanks!

**Comment by astuyve** [comment]
Hi @tibbing - this is fixed the latest release, [v8.107.0](https://github.com/DataDog/datadog-lambda-js/releases/tag/v8.107.0)

v105, v106, and v107 are now available in `eu-north-1`.
![image](https://github.com/DataDog/datadog-lambda-js/assets/1598537/a967ee46-dd75-4459-990f-54068cc0ac0e)

Thanks again!

**Comment by tibbing** [comment]
> Hi @tibbing - this is fixed the latest release, [v8.107.0](https://github.com/DataDog/datadog-lambda-js/releases/tag/v8.107.0)
> 
> v105, v106, and v107 are now available in `eu-north-1`. ![image](https://private-user-images.githubusercontent.com/1598537/311352021-a967ee46-dd75-4459-990f-54068cc0ac0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk5MzQ2NDMsIm5iZiI6MTcwOTkzNDM0MywicGF0aCI6Ii8xNTk4NTM3LzMxMTM1MjAyMS1hOTY3ZWU0Ni1kZDc1LTQ0NTktOTkwZi01NDA2OGNjMGFjMGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMjE0NTQzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2FjZDY4MDRkMzZkNTlkMzExNzQ2NDI5NDIzYzA1ZjE5NjdlZGYzYjI5M2RiNDk1ODEwMDZhY2NlYzA1N2E2NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.BqZNW_JuhbDFFTnZFQVQov17QE7j6VgBFlY86cW4M9c)
> 
> Thanks again!

Awesome, thanks for the quick resolution! 🙌🏻
